src/get-queries-for-element.ts

```diff
@@ -0,0 +1,55 @@
+import * as defaultQueries from './queries'
+
```

Comment:
This area seems the most complex in the migraiton... I know we've gone through a few iterations on the typings for this. Hopefully it doesn't need to change often.